From c6c37d5bd2610dba4ecf65dc9ce5527e042cd2a0 Mon Sep 17 00:00:00 2001
From: "guoyin.chen" <guoyin.chen@freescale.com>
Date: Wed, 8 May 2013 15:48:40 +0800
Subject: [PATCH 197/602] ENGR00261406 Enable data partition encryption
 function for android

Make 32K reserved space for data partition in recovery process

Signed-off-by: guoyin.chen <guoyin.chen@freescale.com>
---
 arm2_6dq/recovery.fstab       | 2 +-
 arm2_6sl/recovery.fstab       | 2 +-
 evk_6sl/recovery.fstab        | 2 +-
 hdmidongle_6dq/recovery.fstab | 2 +-
 sabreauto_6q/recovery.fstab   | 2 +-
 sabresd_6dq/recovery.fstab    | 2 +-
 6 files changed, 6 insertions(+), 6 deletions(-)

diff --git a/arm2_6dq/recovery.fstab b/arm2_6dq/recovery.fstab
index 9c557b8..ef1b961 100644
--- a/arm2_6dq/recovery.fstab
+++ b/arm2_6dq/recovery.fstab
@@ -2,7 +2,7 @@
 /recovery emmc /dev/block/mmcblk0p2
 /system ext4 /dev/block/mmcblk0p5
 /cache ext4 /dev/block/mmcblk0p6
-/data ext4 /dev/block/mmcblk0p4
+/data ext4 /dev/block/mmcblk0p4 reserved=32768
 /misc  emmc /dev/block/mmcblk0p8
 
 # UBI support is under below:
diff --git a/arm2_6sl/recovery.fstab b/arm2_6sl/recovery.fstab
index 97371f6..9f9586c 100644
--- a/arm2_6sl/recovery.fstab
+++ b/arm2_6sl/recovery.fstab
@@ -2,7 +2,7 @@
 /recovery emmc /dev/block/mmcblk0p2
 /system ext4 /dev/block/mmcblk0p5
 /cache ext4 /dev/block/mmcblk0p6
-/data ext4 /dev/block/mmcblk0p4
+/data ext4 /dev/block/mmcblk0p4 reserved=32768
 /misc  emmc  /dev/block/mmcblk0p8
 
 # UBI support is under below:
diff --git a/evk_6sl/recovery.fstab b/evk_6sl/recovery.fstab
index 97371f6..9f9586c 100644
--- a/evk_6sl/recovery.fstab
+++ b/evk_6sl/recovery.fstab
@@ -2,7 +2,7 @@
 /recovery emmc /dev/block/mmcblk0p2
 /system ext4 /dev/block/mmcblk0p5
 /cache ext4 /dev/block/mmcblk0p6
-/data ext4 /dev/block/mmcblk0p4
+/data ext4 /dev/block/mmcblk0p4 reserved=32768
 /misc  emmc  /dev/block/mmcblk0p8
 
 # UBI support is under below:
diff --git a/hdmidongle_6dq/recovery.fstab b/hdmidongle_6dq/recovery.fstab
index cc47236..b2e07c9 100644
--- a/hdmidongle_6dq/recovery.fstab
+++ b/hdmidongle_6dq/recovery.fstab
@@ -4,7 +4,7 @@
 /recovery emmc /dev/block/mmcblk0p2
 /system ext4 /dev/block/mmcblk0p5
 /cache ext4 /dev/block/mmcblk0p6
-/data ext4 /dev/block/mmcblk0p4
+/data ext4 /dev/block/mmcblk0p4 reserved=32768
 /misc  emmc /dev/block/mmcblk0p8
 
 /sdcard vfat /dev/block/mmcblk1p1
diff --git a/sabreauto_6q/recovery.fstab b/sabreauto_6q/recovery.fstab
index 7e44449..a6e5f10 100644
--- a/sabreauto_6q/recovery.fstab
+++ b/sabreauto_6q/recovery.fstab
@@ -2,7 +2,7 @@
 /recovery emmc /dev/block/mmcblk0p2
 /system ext4 /dev/block/mmcblk0p5
 /cache ext4 /dev/block/mmcblk0p6
-/data ext4 /dev/block/mmcblk0p4
+/data ext4 /dev/block/mmcblk0p4 reserved=32768
 /misc  emmc /dev/block/mmcblk0p8
 
 /sdcard vfat /dev/block/mmcblk1p1
diff --git a/sabresd_6dq/recovery.fstab b/sabresd_6dq/recovery.fstab
index 6c86648..84e242b 100644
--- a/sabresd_6dq/recovery.fstab
+++ b/sabresd_6dq/recovery.fstab
@@ -2,7 +2,7 @@
 /recovery emmc /dev/block/mmcblk0p2
 /system ext4 /dev/block/mmcblk0p5
 /cache ext4 /dev/block/mmcblk0p6
-/data ext4 /dev/block/mmcblk0p4
+/data ext4 /dev/block/mmcblk0p4 reserved=32768
 /misc  emmc  /dev/block/mmcblk0p8
 /sdcard vfat /dev/block/mmcblk1p1
 
-- 
1.8.0

